File: eleitor.php
<?php
    $idade = $_GET["idade"];

    if($idade >= 1 and $idade <= 120)
    {
        if($idade < 16)
        {
            echo "Não eleitor";
        }
        elseif($idade >= 18 and $idade <= 65)
        {
            echo "Eleitor obrigatório";
        }
        else
        {
            echo "Eleitor facultativo";
        }
    }
    else
    {
        echo "Idade inválida";
    }

File: Logica de programação/11_03 Terca Exercicio/Python/eleitor.py
idade = int(input("Entre com sua idade : "))
if(idade >= 1 or idade <= 120):
    if(idade < 16):
        print("Não pode votar")
    elif(idade < 18 or idade > 65):
        print("Voto não obrigatório")
    else:
        print("Voto obrigatório")
else:
    print("Idade inválida")
